### PR TITLE
threaded checkor

### PR DESCRIPTION
**Pull request by vlimant**
is this going to help on https://github.com/CMSCompOps/WmAgentScripts/issues/309

### CODE REVIEW

**Comment by thongonary** [comment]
I'm going to merge this by the end of today if there is no objection.

File: Unified/checkor.py [hunk 1]
```diff
@@ -1326,6 +1466,7 @@ def upward( ns ):
     parser.add_option('--tape_size_limit', help='The limit in size of all outputs',default=0,type=int)
     parser.add_option('--html',help='make the monitor page',action='store_true', default=False)
     parser.add_option('--no_report',help='Prevent from making the error report',action='store_true', default=False)
+    parser.add_option('--threads',help='Number of threads for processing workflows',default=1, type=int)
```
Comment (thongonary) [inline Unified/checkor.py]: Is it ok if we set the default to be 5?

Comment (vlimant) [reply Unified/checkor.py]: I have not done any scale or scaling test of this, so any number would do, once robustly tested

Comment (thongonary) [reply Unified/checkor.py]: @SvenjaPflitsch can you please test this on 1 workflow to see if it runs OK with multi threads?

Comment (vlimant) [reply Unified/checkor.py]: you obviously have to try with more than 1 workflow otherwise there is nothing to put on multiple threads ...

**Comment by thongonary** [comment]
I will merge this to test for a day. If something breaks the next day I'll rollback the PR.

**Comment by SvenjaPflitsch** [comment]
I have checked the latest checkor logs and it does not look like there are any big issues. The module is running smoothly at the moment.  12h is a bit early to draw definitive conclusions but for certain options like "review-recovering" things are about 100s faster per workflow. I will keep an eye on things and check the average running time for the different options before and after this pull-request was merged.

**Pull request merged**